app/components/RadioPlayer.vue

```diff
@@ -368,17 +369,25 @@ export default {
       this.is_playing = true
       this.showCurrentMetadata()
 
+      document.title = `🔈 ${this.show_title} - ${this.show_subtitle}`
+      this.docTitleSetter = setInterval(() => {
+        if (this.is_playing) {
+          document.title = `🔈 ${this.show_title} - ${this.show_subtitle}`
+        } else {
+          clearInterval(this.docTitleSetter)
+        }
+      }, 3000)
+
       this.$store.commit('player/isArcsiPlaying', false)
       this.$store.commit('player/isStreamPlaying', true)
 
-      //Google Analytics 4 event: only send if it's a regular show on air
-      if(this.show_check){
+      // Google Analytics 4 event: only send if it's a regular show on air
+      if (this.show_check) {
         gtag('event', 'Radio play', {
-          'Show': this.show_title,
```

Comment:
@baxgas any reason why you removed the apostrophes? Are you sure it would work without them?

Reply:
I think PHPstorm linter autofix might did it. It is ok without, sure it is a js object key, if it is a simple string it is correct like this

Reply:
Ok, let's see.